scheduler/docs/configuration.adoc

```diff
@@ -111,10 +111,6 @@ We'll look at the configurable options in turn:
   This option sets the Mesos master connection string.
   For example, if you are running Mesos with a Zookeeper node on the local machine (a common development setup), you'd use the connection string `zk://localhost:2181/mesos`.
 
-`:master-hosts`::
```

Comment:
Do you think it is worthwhile to have a deprecated or removed section in our config doc so users who upgrade aren't confused why the setting they used to use is no longer documented. Perhaps as an alternative, having the init in the scheduler print that the config is no longer used? 

I don't think it is a big deal so I'm kind of on the fence about it.

Reply:
I like the idea of printing that it's no longer used. I will add that.

Reply:
Done